src/Components/3DV/SceneView.styles.ts

```diff
@@ -55,6 +56,15 @@ export const getSceneViewStyles = memoizeFunction((theme: Theme) =>
                 transform: 'translate(-50%, 0%)'
             } as IStyle
         ],
+        networkErrorMessage: [
```

Comment:
because it was for a different type of error representation, but now made it common as Matt asked to use illustration for the other generic type of errors as well.